docs/source/user_guide/tutorials/getting_started.rst

```diff
@@ -115,623 +392,114 @@ Let's use the alias to print the results parquet, the location of which was prin
-
-.. image:: DAG-e2e-pipeline.svg
-   :width: 500
-
-.. todo::
-   Explain spark in above diagram
-
-
-``e2e/pipeline_expanded.yaml``
-------------------------------
-A longer, more complex pipeline.
-
-.. code-block:: console
-
-   $ easylink run -p specifications/e2e/pipeline_expanded.yaml -i specifications/common/input_data.yaml -e specifications/e2e/environment_slurm.yaml
-   2025-04-01 07:04:16.812 | 0:00:01.500753 | run:158 - Running pipeline
-   2025-04-01 07:04:16.812 | 0:00:01.500984 | run:160 - Results directory: /mnt/share/homes/tylerdy/easylink/tests/results/2025_04_01_07_04_16
-   ...
-   [Tue Apr  1 07:27:22 2025]
-   Job 0: Grabbing final output
-   Reason: Missing output files: result.parquet; Input files updated by another job: intermediate/step_4b_python_pandas/result.parquet, input_validations/final_validator
-
-
-.. code-block:: console
-
-   $ pqprint results/2025_04_01_07_04_16/result.parquet
-            foo bar  counter  added_column_2  added_column_3  added_column_4  added_column_5  added_column_6
-   0          0   a        6             2.0             3.0             4.0             5.0               6
-   1          1   b        6             2.0             3.0             4.0             5.0               6
-   2          2   c        6             2.0             3.0             4.0             5.0               6
-   3          3   d        6             2.0             3.0             4.0             5.0               6
-   4          4   e        6             2.0             3.0             4.0             5.0               6
-   ...      ...  ..      ...             ...             ...             ...             ...             ...
-   149995  9995   a        1             0.0             0.0             0.0             0.0               6
-   149996  9996   b        1             0.0             0.0             0.0             0.0               6
-   149997  9997   c        1             0.0             0.0             0.0             0.0               6
-   149998  9998   d        1             0.0             0.0             0.0             0.0               6
-   149999  9999   e        1             0.0             0.0             0.0             0.0               6
-
-   [150000 rows x 8 columns]
-
-.. image:: DAG-e2e-pipeline-expanded.svg
-   :width: 600
-
-
-That's all the valid pipelines currently available in the ``tests`` directory! Next we will create
-some pipelines of our own to run by copying the ``tests`` pipelines and making some changes.
\ No newline at end of file
+Finally, lets run this same "improved" pipeline, but using :download:`input_data_demo_2030.yaml` 
+as the input YAML, which uses the ``SSA`` and ``W2`` datasets from ``2030`` rather than 
+``2020``::
+
+    $ easylink run -p pipeline_demo_improved_2030.yaml -i input_data_demo_2030.yaml -e environment_local.yaml -I /mnt/team/simulation_science/priv/engineering/er_ecosystem/images
+    
+For a threshold match
```

Comment:
Insert a paragraph before this one (once it's true!) saying the utility of this, i.e. that you can run the same pipeline on different data by changing only one parameter